articles/libraries/machine-learning/basic-classification.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Quantum machine learning library
+author: geduardo
```

Comment:
description: Learn how to execute a quantum sequential classifier written in Q# using the Quantum Machine Learning library of the Microsoft Quantum Development Kit.